### ISSUE TITLE
Add Close (X) button to modal ui

### ISSUE THREAD

**Issue by Aluisyo** [script3/yieldblox-beta#101]
When a pop up modal comes up, there is now close button instead user has to click on background interfafe to dismiss modal. This can be hard to use when using a mobile device and can be a bit confusing for a new user since they wouldnt know how to close the modal.

A close button would be the solution to this.

![borrowmodal](https://user-images.githubusercontent.com/11541812/136327304-4d3122a4-4f54-41e4-ae3d-30785ace43a9.png)
![claimmodal](https://user-images.githubusercontent.com/11541812/136327311-d6748766-c3b6-4a32-ae9e-5e6f05e1973b.png)
![stkmodal](https://user-images.githubusercontent.com/11541812/136327314-59896278-7115-4c71-9a26-350acc5972a4.png)
![ithdrmodal](https://user-images.githubusercontent.com/11541812/136327319-3012da04-ce69-481e-9201-c1d3873bfe3c.png)
![updtmodl](https://user-images.githubusercontent.com/11541812/136327324-ade08b8a-1a30-4e24-bd54-00b31dc46a9b.png)
![errormodal](https://user-images.githubusercontent.com/11541812/136327330-e3fe0f33-68ed-44a4-9310-7d01df339fff.png)
![deposit modal](https://user-images.githubusercontent.com/11541812/136327334-5cb6890e-7351-49a9-9e7f-7e11cf142597.png)

**Comment by mootz12** [comment]
Yeah, that is on the list of things to do. Thanks for the suggestion!
